Examples/Tests/electrostatic_sphere_eb/inputs_3d

```diff
@@ -17,7 +17,7 @@ geometry.prob_hi     =  0.5  0.5  0.5
 warpx.const_dt = 1e-6
 
 warpx.do_electrostatic = labframe
-warpx.eb_implicit_function = "-(x**2+y**2+z**2-0.3**2)"
+warpx.eb_implicit_function = "-(x**2+y**2+z**2-0.1**2)"
```

Comment:
I changed the radius to `0.1` because otherwise the EB was too close to the boundaries of the simulation box, and the results were therefore significantly different from the theoretical expected results.